### DISCUSSION TITLE
Flexible sensor multiplication factor

### DISCUSSION THREAD

**Discussion by Jens5395** [emporia-vue-local/esphome#35] [Q&A]
I am hooking up two flexible sensors to my vue2 but they register at only a fraction of what they should register at. I am assuming that I would need to calibrate them with a known load for optimum accuracy. I am wondering if anyone is using flexible current sensors and if so, what kind of a calibration factor are you using.
If it makes any difference, I am using these sensors on the smaller ports and not in one of the three large ports on the top of the vue2 Via an adapter to go from 3.5 mm to 2.5 mm plug).
I would also like to confirm that the way these are calibrated is via the multiplication factor setting in the individual current clamp sensor configuration line ie:

```
 - { phase_id: phase_b, input:  "8", power: { name:  "Circuit 8 Power", id:  cir8, filters: [ *moving_avg, *abs, multiply: x.xx ] } }
 ```

**Comment by flaviut** [comment]
> I am assuming that I would need to calibrate them with a known load for optimum accuracy.
> ...
>  I would also like to confirm that the way these are calibrated is via the multiplication factor setting in the individual current clamp sensor configuration line ie:

You are exactly right on both things!

I'm not aware of anyone else using these, but please report back with with the correct calibration value. I'm sure others will find it very helpful.
